### PR TITLE
Edit profile page

### PR DESCRIPTION
**Pull request by segirim**
- Added autosave for prettier.
- Some refactor.
- Added Edit profile page. Will refactor Edit profile page later into multiple components.

### CODE REVIEW

File: src/pages/404/404.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import Layout from 'components/Layout'
 
 export const NoMatch = () => {
     return (
-        <Layout boxed>
+        <Layout>
```
Comment (nopestack) [inline src/pages/404/404.tsx:10]: Why was `boxed` removed with no apparent change to the page's structure?

File: ownemployed.code-workspace [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (nopestack) [inline ownemployed.code-workspace]: This file seems to be VSCode specific and not necessary for the application to run. Would you remove it?

**Review by nopestack — CHANGES_REQUESTED** [review]
Overall great work. I'm curious as to why the boxed layout is not desired in the pages it was removed? I recall needing to have a fullwidth layout on `home.tsx` because the spec required fullwidth sections in there, but the mockups seem to maintain a consistent content width limit across all other pages.

File: src/pages/404/404.tsx (hunk 1 above)
Comment (segirim) [reply src/pages/404/404.tsx:10]: boxed-attribute forced Main component to have "margin: auto". That destroys the Main child if they don't have a fixed width or child with elements that takes the whole width.

File: ownemployed.code-workspace (hunk 2 above)
Comment (segirim) [reply ownemployed.code-workspace]: Yes, can remove it and add it to .gitignore.

**Review by nopestack — APPROVED** [review]

**Pull request merged**